xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/api/XWiki.java

```diff
@@ -57,6 +57,8 @@
 
 public class XWiki extends Api
 {
+    private static final int MAIL_SENDER_WRONGLY_CONFIGURED = -11;
```

Comment:
Checkstyle will complain, `private` field must come after the `protected` ones.

And I will complain that this is the first such error defined here, and there's a good reason why other haven't been defined so far: it's not the right place. This is a thin wrapper that shouldn't fix problems in the underlying class. Why not return the right error code from the privileged `com.xpn.xwiki.XWiki` class instead?